winui/Avatar-view/visual-styles.md

```diff
@@ -1,10 +1,404 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: This section will provide an explanation of the various visual styles that are available in WinUI AvatarView.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Visual Styles in WinUI AvatarView
\ No newline at end of file
+# Visual Styles in WinUI AvatarView
+
+`AvatarView` control allows for customization of visual styles by providing a range of built-in styles. You can configure the visual style of your AvatarView by using the `AvatarShape` property and select from the available options, such as:
+
+* Custom
+* Circle
+* Square
+
+## Custom
+
+Custom type enables extensive customization options such as adjusting the size, font size, corner radius, and other properties.
```

Comment:
@MadhumithanJayapathi Custom type allows you to customize the AvatarView control with height, width, font and corner radius.

Reply:
@Manivannan-E Changed the content of custom as mentioned.